pkg/cmd/install.go

```diff
@@ -71,12 +74,15 @@ It's possible to source information about hosts from Terraform output, using the
 
 	cmd.Flags().StringVarP(&iopts.BackupFile, "backup", "b", "", "path to where the PKI backup .tar.gz file should be placed (default: location of cluster config file)")
 
+	cmd.Flags().StringVarP(&iopts.AWSProfilePath, "aws-profile-path", "", "~/.aws/credentials", "path to the file where aws credentials are located")
+	cmd.Flags().StringVarP(&iopts.AWSProfileName, "aws-profile-name", "", "default", "name of the aws profile to be deployed on the cluster for machine-controller")
```

Comment:
I suppose this flag should be empty by default, so people would have to explicitly opt-in and provide `--aws-profile-name=whatever` in order to give-up credentials from `~/.aws/credentials`.